tournamentcontrol/competition/admin.py

```diff
@@ -408,6 +414,11 @@ def get_urls(self):
                     "<int:season_id>/division/",
                     include(division_urls, namespace="division"),
                 ),
+                path(
+                    "<int:season_id>/bulk-divisions/",
+                    self.bulk_divisions,
+                    name="bulk-divisions",
```

Comment:
The view should be surfaced in the `division_urls`, more like the `edit_division` view. It should be at `bulk/` and named `bulk` rather than `bulk-divisions` as it will be namespaced.